app/js/OptionsTab.js

```diff
@@ -427,6 +428,21 @@ class OptionsTab extends React.Component<OptionsTabProps, OptionsTabState> {
               {chrome.i18n.getMessage("options_option_filterAudio_label")}
             </label>
           </div>
+          {BROWSER === "chrome" && (
```

Comment:
I misled you on this, it would be best to feature check rather than match on a specific browser. If Firefox implements the same feature in the future, Tab Wrangler would magically start working there.

This could get the current tab and see if `groupID` is in it, the same check you added in tab.js:13.